README.md

```diff
@@ -146,7 +146,7 @@ Devika requires certain configuration settings and API keys to function properly
 
 - `OPENAI_API_KEY`: Your OpenAI API key for accessing GPT models.
 - `CLAUDE_API_KEY`: Your Anthropic API key for accessing Claude models.
-- `BING_API_KEY`: Your Bing Search API key for web searching capabilities.
+- `SERP_API_KEY`: Your SERP Search API key for web searching capabilities.
```

Comment:
This should be configurable instead of just removing bing. In other PRs people also adds/suggests google, DDGS, others, so a better approach is coexistence

Reply:
With SerpAPI you can choose which search engine you want to use (e. g. google, bing, google, duckduckgo etc.)

Reply:
I was checking and the pricing model is restraining for serpapi

References

https://dataforseo.com/pricing

https://www.microsoft.com/en-us/bing/apis/pricing